substrate/frame/membership/src/tests.rs

```diff
@@ -0,0 +1,225 @@
+// This file is part of Substrate.
```

Comment:
The files are never used?

Reply:
Had missed this.
Added these files into `lib.rs`.